tls/s2n_signature_scheme.c

```diff
@@ -17,88 +17,93 @@
 
 #include "crypto/s2n_hash.h"
 #include "crypto/s2n_signature.h"
+#include "tls/s2n_connection.h"
 #include "tls/s2n_signature_scheme.h"
 #include "crypto/s2n_ecc_evp.h"
 #include "utils/s2n_safety.h"
 
-
 /* RSA PKCS1 */
 const struct s2n_signature_scheme s2n_rsa_pkcs1_md5_sha1 = {
         .iana_value = TLS_SIGNATURE_SCHEME_PRIVATE_INTERNAL_RSA_PKCS1_MD5_SHA1,
         .hash_alg = S2N_HASH_MD5_SHA1,
         .sig_alg = S2N_SIGNATURE_RSA,
-        .signature_curve = NULL /* Elliptic Curve not needed for RSA */
+        .signature_curve = NULL, /* Elliptic Curve not needed for RSA */
 };
 
 const struct s2n_signature_scheme s2n_rsa_pkcs1_sha1 = {
         .iana_value = TLS_SIGNATURE_SCHEME_RSA_PKCS1_SHA1,
         .hash_alg = S2N_HASH_SHA1,
         .sig_alg = S2N_SIGNATURE_RSA,
-        .signature_curve = NULL /* Elliptic Curve not needed for RSA */
+        .signature_curve = NULL, /* Elliptic Curve not needed for RSA */
 };
 
 const struct s2n_signature_scheme s2n_rsa_pkcs1_sha224 = {
         .iana_value = TLS_SIGNATURE_SCHEME_RSA_PKCS1_SHA224,
         .hash_alg = S2N_HASH_SHA224,
         .sig_alg = S2N_SIGNATURE_RSA,
-        .signature_curve = NULL /* Elliptic Curve not needed for RSA */
+        .signature_curve = NULL, /* Elliptic Curve not needed for RSA */
+        .maximum_protocol_version = S2N_TLS12,
```

Comment:
explicitly set this on all?

Reply:
Idk about explicitly setting max. For most signature schemes, if we add another protocol version, they will still be valid. Updating the max for all schemes feels less clear than just adding a max for those that are no longer supported.

Reply:
discussed offline.  i'm find with leaving it as is, if it turns out someone has issues without being explicit on all this, we can address. on the plus side, you have conviniently added the commas in place already :)